app/components/HomeContent.js

```diff
@@ -8,7 +8,7 @@ export default class HomeContent extends Component {
     const isAtLeastOneNote = notes.length > 0
 
     // Show homeContent when no note is selected.
-    if (this.props.activeNoteIndex >= 0) { return (<NoteEditor />) }
+    if (this.props.activeNoteIndex >= 0 && this.notes[this.props.activeNoteIndex]) { return (<NoteEditor />) }
```

Comment:
Use notes directly instead of this.notes and keep line-length around 80 chars.
```suggestion
     // Show homeContent when no note is selected.
    if (this.props.activeNoteIndex >= 0 &&
        notes[this.props.activeNoteIndex]) {
      return (<NoteEditor />)
    }
```